lib/peep.rb

```diff
@@ -0,0 +1,10 @@
+require 'pg'
+
+class Peep
+
+  def self.all 
+    peeps = DatabaseConnection.connect.exec("SELECT * FROM peeps")
+    peeps.map { |peep| peep['message'] }
```

Comment:
If you were to carry on could use .reverse to show them in reverse chronological order